br/pkg/gluetidb/glue.go

```diff
@@ -70,10 +70,6 @@ func (g Glue) GetDomain(store kv.Storage) (*domain.Domain, error) {
 	if err != nil {
 		return nil, errors.Trace(err)
 	}
-	se, err := g.createTypesSession(store)
```

Comment:
@Leavrth Could you please take a look? We didn't use this session at all. So I deleted it.